samples/drivers/w1/scanner/sample.yaml

```
@@ -50,3 +50,18 @@ tests:
       regex:
         - "Number of devices found on bus: .*"
       fixture: w1_scanner_w1_serial
+  samples.drivers.w1.scanner.max:
```

Nit. Could you additionally rename the test configuration as suggested in #85629

```suggestion
  sample.drivers.w1.scanner.max:
```

---

Done. Thanks for warning.